apache-activemq-mixin/dashboards/apache-activemq-queue-overview.libsonnet

```diff
@@ -0,0 +1,877 @@
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'mps',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local topQueuesByDequeueRatePanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'topk by(instance, activemq_cluster) ($k_selector, rate(activemq_queue_dequeue_count{' + matcher + ',instance=~"$instance", destination=~".*$name.*"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{activemq_cluster}} - {{instance}} - {{destination}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Top queues by dequeue rate',
+  description: 'The rate messages have been acknowledged (and removed) from the top k queue destinations.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        axisShow: false,
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 25,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'mps',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local topQueuesByAverageEnqueueTimePanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'topk by(instance, activemq_cluster) ($k_selector, activemq_queue_average_enqueue_time{' + matcher + ', instance=~"$instance", destination=~".*$name.*"})',
+      datasource=promDatasource,
+      legendFormat='{{activemq_cluster}} - {{instance}} - {{destination}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Top queues by average enqueue time',
+  description: 'Average time a message was held on top k queue destinations.',
```

Comment:
Minor: won't hold up over this but i think i would usually leave out "top k" wording form definitions. "Average time messages were held for the..."